integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/person/PersonEntityResource.java

```diff
@@ -40,6 +41,15 @@ public void addPersons(List<PersonEntity> persons) {
         PersonEntity.persist(persons);
     }
 
+    // This endpoint is not called inside the test because transaction needs a replica set
```

Comment:
Really? That makes it pretty hard to test transactions. I don't see why either…

Reply:
Transactions only works on a replicat set, MongoDB only provides distributed treansaction, that's why.
I'll investiguate a way to setup a cluster via Quakus TestResource support.

Reply:
I setup a replica set for the integration test. I copied the existing TestResource from the `mongob-client` extension.
Maybe I need to create an `quarkus-test-mongodb` module inside the `test-framework` directory and put the MongoDB related test resource in it to avoid duplication ?